pkg/autometrics/helpers.go

```diff
@@ -0,0 +1,41 @@
+package autometrics // import "github.com/autometrics-dev/autometrics-go/pkg/autometrics"
+
+import (
+	"errors"
+	"fmt"
+	"net"
+
+	"github.com/oklog/ulid/v2"
+)
+
+// GetOutboundIP returns the preferred outbound ip of this machine
+//
+// This is a useful function to use for setting a `job` key when
+// pushing metrics
+func GetOutboundIP() (net.IP, error) {
+	conn, err := net.Dial("tcp", "1.1.1.1:80")
```

Comment:
eehh this may be nasty behavior imo, it is quite unexpected for the autometrics library to make connections to the wide internet. definitely need to document this behavior more obviously

Reply:
Fair enough